src/views/FSL/lookup/lookup.vue

```diff
@@ -1877,6 +1906,14 @@ export default {
       return $(`#amortization`).modal("toggle")
     },
 
+    displayDebitCard () {
```

Comment:
can do this here

`displayDebitCard() {
  const allowedStatuses = ['Completed', 'Repossessed', 'Closed'];
  return !allowedStatuses.includes(this.order.status);
}
`